sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -94,9 +98,32 @@ internal object RadarSettings {
     }
 
     internal fun getTrackingOptions(context: Context): RadarTrackingOptions {
-        val optionsJson = getSharedPreferences(context).getString(KEY_TRACKING_OPTIONS, null) ?: return RadarTrackingOptions.EFFICIENT
-        val optionsObj = JSONObject(optionsJson)
-        return RadarTrackingOptions.fromJson(optionsObj)
+        val optionsJson = getSharedPreferences(context).getString(KEY_TRACKING_OPTIONS, null)
+        val options: RadarTrackingOptions?
+        if (optionsJson != null) { // v3 tracking options set
+            val optionsObj = JSONObject(optionsJson)
+            options = RadarTrackingOptions.fromJson(optionsObj)
+        } else {
+            val oldInterval = getSharedPreferences(context).getInt(KEY_OLD_UPDATE_INTERVAL, 0)
+            if (oldInterval > 0) { // v2 tracking options upgrade
+                options = if (oldInterval == 60000) {
```

Comment:
define this as a const somewhere? - `RESPONSIVE_INTERVAL_V2`

Reply:
Will do